src/en/charms-storage.md

```diff
@@ -386,19 +412,21 @@ provider currently supports a single pool configuration attribute:
     - 'oracle' (volume type is 'default')
     - 'oracle-latency' (volume type is 'latency').
 
+### LXD (lxd)
+
```

Comment:
Yep, that's coming in the next PR. I have some questions. :speech_balloon: Will send email.